### DISCUSSION TITLE
Something woring in the docs?

### DISCUSSION THREAD

**Discussion by arnelamo** [supabase/supabase#28262] [Questions]
When I try to setup my serverClient like described in the [docs](https://supabase.com/docs/guides/auth/server-side/nextjs?queryGroups=router&router=app), I get this TS error:

Object literal may only specify known properties, and 'getAll' does not exist in type 'CookieMethods'.

**Comment by yudistiraashadi** [comment]
You need to use their latest version. I'm using 0.4.0 and the error is gone
